src/app/pages/BuyPage/components/Button/close.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled, { css } from 'styled-components/macro';
```

Comment:
Pls remvoe this line, we no longer need it.

Reply:
fixed